gensrc/thrift/FrontendService.thrift

```diff
@@ -443,6 +443,7 @@ struct TMasterOpRequest {
     15: optional i32 stmtIdx  // the idx of the sql in multi statements
     16: optional PaloInternalService.TQueryOptions query_options
     17: optional Types.TUniqueId query_id // when this is a query, we translate this query id to master
+    18: optional i64 insert_visible_timeout_ms
```

Comment:
Should it be set in MasterOpExecutor?

Reply:
We also can add a hint `insert /*+ SET_VAR(insert_visible_timeout_ms = 20000)*/ xxx` like select hint `select /*+ SET_VAR */ ` and transform this sql with hint to master.
Any other better place?